jcache/embedded/src/main/java/org/infinispan/jcache/embedded/JCache.java

```diff
@@ -371,7 +371,9 @@ public T call() throws Exception {
    @Override
    public <T> Map<K, EntryProcessorResult<T>> invokeAll(
          Set<? extends K> keys, EntryProcessor<K, V, T> entryProcessor, Object... arguments) {
-      checkNotClosed().checkNull(entryProcessor, "entryProcessor").verifyKeys(keys);
```

Comment:
I moved the null check to the superclass to avoid duplication and I'm planning to do the same for the other checks. I don't think it's required for them to be fluent. They could still be placed on a single line if it's easier to read that way.